perception/ground_segmentation/src/scan_ground_filter_nodelet.cpp

```diff
@@ -58,6 +58,11 @@ ScanGroundFilterComponent::ScanGroundFilterComponent(const rclcpp::NodeOptions &
     use_virtual_ground_point_ = declare_parameter<bool>("use_virtual_ground_point");
     use_recheck_ground_cluster_ = declare_parameter<bool>("use_recheck_ground_cluster");
     use_lowest_point_ = declare_parameter<bool>("use_lowest_point");
+    if (declare_parameter<bool>("output_intensity_field")) {
```

Comment:
[imo]
We should check the input pointcloud topic instead of making a new parameter?

Reply:
@miursh I reverted the parameter adding commit here https://github.com/autowarefoundation/autoware.universe/pull/6791/commits/13c76f1f07bb0d071c4a67caba3b04a55ec1c42f
and copy the input topic field to output at https://github.com/autowarefoundation/autoware.universe/pull/6791/commits/0ee96121570bb002b19b1a6dfda2bb4e3b1ff81d .